test/onnx/exporter/test_small_models_e2e.py

```diff
@@ -736,11 +748,17 @@ def forward(self, query, key, value):
         query = torch.rand(32, 8, 128, 64, dtype=torch.float16)
         key = torch.rand(32, 8, 128, 64, dtype=torch.float16)
         value = torch.rand(32, 8, 128, 64, dtype=torch.float16)
+        expected = Model()(query, key, value)
 
         onnx_program = self.export(Model(), (query, key, value), opset_version=23)
         self.assertIn("Attention", [node.op_type for node in onnx_program.model.graph])
 
-    @pytest.mark.xfail(reason="Expected to fail until opset 23 is supported by ORT.")
+        ref = onnx_ref.ReferenceEvaluator(onnx_program.model_proto)
+        got = ref.run(
+            None, dict(query=query.numpy(), key=key.numpy(), value=value.numpy())
+        )[0]
+        torch.testing.assert_close(expected, torch.from_numpy(got))
```

Comment:
```suggestion
        torch.testing.assert_close(torch.from_numpy(got), expected)
```

the assert function will display the second argument as expected if there is a mismatch